assets/xml/objects/object_zov.xml

```diff
@@ -1,81 +1,82 @@
+        <DList Name="gLuluTorsoDL" Offset="0x107B0" />
+        <DList Name="gLuluRightUpperArmDL" Offset="0x10BC0" />
+        <DList Name="gLuluRightForearmDL" Offset="0x10E60" />
+        <DList Name="gLuluRightHandDL" Offset="0x10FF0" />
+        <DList Name="gLuluRightArmUpperFinDL" Offset="0x11230" />
+        <DList Name="gLuluRightArmLowerFinDL" Offset="0x11300" />
+        <DList Name="gLuluLeftUpperArmDL" Offset="0x113F0" />
+        <DList Name="gLuluLeftForearmDL" Offset="0x11690" />
+        <DList Name="gLuluLeftHandDL" Offset="0x11820" />
+        <DList Name="gLuluLeftArmUpperFinDL" Offset="0x11A60" />
+        <DList Name="gLuluLeftArmLowerFinDL" Offset="0x11B30" />
+        <DList Name="gLuluHeadDL" Offset="0x11C20" />
+        <DList Name="gLuluPelvisDL" Offset="0x122E0" />
+        <DList Name="gLuluRightThighDL" Offset="0x123C0" />
+        <DList Name="gLuluRightShinDL" Offset="0x12560" />
+        <DList Name="gLuluRightLegFinDL" Offset="0x127D0" />
+        <DList Name="gLuluRightFootDL" Offset="0x12890" />
+        <DList Name="gLuluLeftThighDL" Offset="0x12B20" />
+        <DList Name="gLuluLeftShinDL" Offset="0x12CB8" />
+        <DList Name="gLuluLeftLegFinDL" Offset="0x12F28" />
+        <DList Name="gLuluLeftFootDL" Offset="0x12FE8" />
+        <Texture Name="gLuluTLUT" OutName="lulu_tlut" Format="rgba16" Width="16" Height="16" Offset="0x13278" />
+        <Texture Name="gLuluEarringTex" OutName="lulu_earring" Format="rgba16" Width="8" Height="16" Offset="0x13478" />
+        <Texture Name="gLuluSkinTex" OutName="lulu_skin" Format="ci8" Width="8" Height="8" Offset="0x13578" />
+        <Texture Name="gLuluNostrilTex" OutName="lulu_nostril" Format="ci8" Width="8" Height="8" Offset="0x135B8" />
+        <Texture Name="gLuluMouthClosedTex" OutName="lulu_mouth_closed" Format="ci8" Width="32" Height="32" Offset="0x135F8" />
+        <Texture Name="gLuluHeadSkinTex" OutName="lulu_head_skin" Format="ci8" Width="16" Height="16" Offset="0x139F8" />
+        <Texture Name="gLuluEarCavityTex" OutName="lulu_ear_cavity" Format="ci8" Width="8" Height="8" Offset="0x13AF8" />
+        <Texture Name="gLuluHeadHoleTex" OutName="lulu_head_hole" Format="ci8" Width="16" Height="16" Offset="0x13B38" />
+        <Texture Name="gLuluEyeOpenTex" OutName="lulu_eye_open" Format="ci8" Width="32" Height="32" Offset="0x13C38" />
+        <Texture Name="gLuluEarTex" OutName="lulu_ear" Format="ci8" Width="16" Height="16" Offset="0x14038" />
+        <Texture Name="gLuluEyeClosedTex" OutName="lulu_eye_closed" Format="ci8" Width="32" Height="32" Offset="0x14138" />
+        <Texture Name="gLuluMouthOpenTex" OutName="lulu_mouth_open" Format="ci8" Width="32" Height="32" Offset="0x14538" />
+        <Texture Name="gLuluSpottedSkinTex" OutName="lulu_spotted_skin" Format="ci8" Width="32" Height="32" Offset="0x14938" />
+        <Texture Name="gLuluMouthUnusedTex" OutName="lulu_mouth_unused" Format="ci8" Width="32" Height="32" Offset="0x14D38" />
```

Comment:
Looking at the texture, perhaps we could name it something like `LuluMouth` +  `OpenWide`/`OpenShocked`/`OpenFull` and just add a comment that it is unused.